packages/testing/README.md

```diff
@@ -71,7 +71,7 @@ Already part of `yo open-wc:testing`
   },
   ```
 
-For more details pls see [testing-karma](https://open-wc.org/recommendations/testing-karma.html)
+For more details pls see [testing-karma](https://open-wc.org/testing/testing-karma.html)
```

Comment:
For more details, please see [testing-karma](https://open-wc.org/testing/testing-karma.html)